### DISCUSSION TITLE
Matematik yarışması sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#262] [Soru Cevap]
Sonsuzusun düzenlediği bir matematik yarışmasına 81 ilden (her ilden en az 1 en çok 10 öğrenci olmak üzere) bilinmeyen sayıda öğrenciler katılmıştır. Bütün öğrencilere yarışma salonuna girme sıralarına göre 1 ‘den başlamak üzere sırayla giriş numaraları verilmiştir. (Salona ilk girene 1, İkinci girene 2, vb.) Yarışma sonunda birinciliği kazanan öğrenci ye giriş numarası sorulduğunda şu cevabı vermiştir:
“Salona benden önce gelen arkadaşların giriş numaralarının toplamıyla benden sonra gelenlerin giriş numaralarının toplamı birbirine eşittir. Benim numaramı buradan bulabilirsiniz.”
Öğrencinin giriş numarasını bulabilir misiniz?

**Comment by DeniZZece** [comment]
```
#include<stdio.h>
int fonksiyon(a,b)
{
	int i,toplam=0;
	for (i=a+1;i<b;i++) toplam+=i;
	return toplam; 
}
int main()
{
	int katilan,kisi;
	for (katilan=81;katilan<=810;katilan++)
	{
		for (kisi=katilan/2;kisi<katilan;kisi++)
		{
			if(fonksiyon(0,kisi)==fonksiyon(kisi,katilan+1))
			{
				printf("%d",kisi); return 1;
			}
		}
	}
}
```
